package.json

```diff
@@ -250,7 +252,7 @@
     "@babel/plugin-proposal-class-properties": "^7.16.7",
     "@babel/plugin-proposal-private-methods": "^7.14.5",
     "@babel/plugin-proposal-private-property-in-object": "^7.15.4",
-    "@babel/plugin-syntax-dynamic-import": "^7.8.3",
+    "@babel/preset-env": "^7.16.11",
```

Comment:
please revert this

- we're using plugin-syntax-dynamic-import 
- we also have "babel-preset-env": "^1.7.0"

last time we tried to update those packages we've meth with some issues so we had to revert the update
let's do not touch them if this doesn't needed strictly

Reply:
I'll revert them must have happened by mistake !